engine/access/rpc/engine.go

```diff
@@ -166,15 +169,15 @@ func New(log zerolog.Logger,
 		chain:              chainID.Chain(),
 	}
 
-	accessproto.RegisterAccessAPIServer(
-		eng.unsecureGrpcServer,
-		access.NewHandler(backend, chainID.Chain()),
-	)
+	var LocalAPIServer accessproto.AccessAPIServer
+	LocalAPIServer = access.NewHandler(backend, chainID.Chain())
+	if proxy != nil {
+		proxy.AccessAPIServer = LocalAPIServer
```

Comment:
Fix: https://github.com/GetElastech/flow-go/commit/99d8d9262319c579f14eecc580c32c495e62cbff

Reply:
what I mean here is to add a `SetLocalAPI(accessproto.AccessAPIServer)` which sets the embedded `AccessAPIServer` interface. I think it will be confusing to set an embedded interface on the proxy within a separate file. If you put it into an explicit setting, you can add some context with comments about what's happening any why we've done it this way.

Reply:
Fix: https://github.com/GetElastech/flow-go/commit/aa9b2c4b25d758f88340ec043516e84ba3b8c33b

```
func (h *FlowAPIService) SetLocalAPI(local access.AccessAPIServer) {
        h.AccessAPIServer = local
}
```